logger/logger.go

```
@@ -12,104 +12,129 @@ import (
-	Logw         = Logf
-	logw         = Logf
-	logf         = Logf
-	logFile      *os.File
-	logger       *log.Logger
-	logChannel   = make(chan logMessage, defaultBufSize)
-	quitChannel  = make(chan struct{})
-	logFileMutex sync.Mutex
-	wg           sync.WaitGroup
-	logLevel     = LevelDump // 默认日志等级为 Dump
+	Logw         = Logf                                                        // 快捷方式
+	logw         = Logf                                                        // 快捷方式
```

golangci-lint (1.62.2)

var `logw` is unused

(unused)